<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewWordDialog</class>
 <widget class="QDialog" name="AddNewWordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Words</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="addWordGroupBox">
     <property name="title">
      <string>Add word</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="addRadioButton">
          <property name="text">
           <string>Add manually</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="readRadioButton">
          <property name="text">
           <string>Read from file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="setLabel">
          <property name="text">
           <string>Set of words</string>
          </property>
          <property name="buddy">
           <cstring>setComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="setComboBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addSetBtn">
          <property name="text">
           <string>Add set to preview</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="wordLabel">
          <property name="text">
           <string>Word:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="languageLabel">
          <property name="text">
           <string>Language:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="originalLabel">
          <property name="text">
           <string>Original:</string>
          </property>
          <property name="buddy">
           <cstring>originalLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="originalLineEdit"/>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="languageOriginalComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="translatedLabel">
          <property name="text">
           <string>Translated:</string>
          </property>
          <property name="buddy">
           <cstring>translatedLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="translatedLineEdit"/>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="languageTranslatedComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>browse</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="filePathLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="fileNameLabel">
          <property name="text">
           <string>File path:</string>
          </property>
          <property name="buddy">
           <cstring>filePathLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="fromLabel">
          <property name="text">
           <string>from...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="intoLabel">
          <property name="text">
           <string>into...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add word to preview</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="previewView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="autosavingCheckBox">
          <property name="toolTip">
           <string>Save preview to data base each time when set or new word added</string>
          </property>
          <property name="text">
           <string>Autosaving</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>Save to Data Base</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addRadioButton</tabstop>
  <tabstop>readRadioButton</tabstop>
  <tabstop>setComboBox</tabstop>
  <tabstop>addSetBtn</tabstop>
  <tabstop>originalLineEdit</tabstop>
  <tabstop>translatedLineEdit</tabstop>
  <tabstop>languageOriginalComboBox</tabstop>
  <tabstop>languageTranslatedComboBox</tabstop>
  <tabstop>filePathLineEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>previewView</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>closeBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddNewWordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
